### PR TITLE
Clarify requirements on RHEL managed hosts

### PR DESCRIPTION
**Pull request by pcahyna**

### CODE REVIEW

**Review by kurik — APPROVED** [review]
Thanks Pavel. Looks good to me.

File: README.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ by the managed host.
 
 Uses features of Performance Co-Pilot (PCP) v5+, Redis v5+ and Grafana v6+.
 However, use of Grafana and Redis is optional (disabled by default).
+The `metrics_graph_service` and `metrics_query_service` options are thus
+supported on RHEL/CentOS starting from major version 8, because
+earlier versions do not have the Grafana and Redis packages. (On
+earlier versions you may enable EPEL repositories as a workaround, but
```
Comment (natoscott) [inline README.md:14]: @pcahyna looks good.  I would drop the bracketed sentence though - Grafana doesn't exist even in EPEL, and the version of Redis in both EPEL6 and EPEL7 is too old for our needs.

File: README.md [hunk 2]
```diff
@@ -8,6 +8,15 @@ by the managed host.
...
+the role is not guaranteed to work with EPEL packages, as this case is
+not tested enough.)
+
+On RHEL 7 and RHEL 6 it is needed to have the Optional repository
```
Comment (natoscott) [inline README.md:18]: So I'd also drop this last sentence - its misleading - we should just assert RHEL8 (and Fedora) are the best sources of up-to-date, working packages for the graphing and scalable querying options.

**Review by natoscott — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pcahyna) [reply README.md:14]: ok

File: README.md (hunk 2 above)
Comment (pcahyna) [reply README.md:18]: The last sentence does not relate to graphing and scalable querying options. Optional is needed for the basic functionality. #17 was not sufficient, there are other packages available only in Optional (I think `pcp-pmda-dm`).

Comment (natoscott) [reply README.md:18]: @pcahyna Ah, I see what you mean - that final sentence looks good as-is then.

File: README.md (hunk 1 above)
Comment (natoscott) [reply README.md:14]: @richm Centos is in the same broader category as RHEL - we could perhaps use words like "... on Fedora, CentOS8, and RHEL8"  (explicitly recommending el8 only in both cases, whereas any supported Fedora is fine)

**Pull request merged**